src/DCSAM.cpp

```diff
@@ -30,8 +30,18 @@ void DCSAM::update(const gtsam::NonlinearFactorGraph &graph,
                    const gtsam::DiscreteFactorGraph &dfg,
                    const DCFactorGraph &dcfg,
                    const gtsam::Values &initialGuessContinuous,
-                   const DiscreteValues &initialGuessDiscrete) {
-  // First things first: combine currContinuous_ estimate with the new values
+                   const DiscreteValues &initialGuessDiscrete,
+                   const gtsam::FactorIndices &removeFactorIndices,
```

Comment:
A trivial thing is can we just pass in the factor indices for the continuous graph, by maybe passing a default empty vector to discrete indices?

Reply:
I believe what you're asking for is already implemented with the default arguments written out in the DCSAM.h file. Please let me know if that's not what you meant though!

Reply:
Ah cool. I think the current impl looks good, like we can remove continuous and discrete factors separately